File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        int slow = 0;
        int fast = 1;

        while(fast < nums.Length){
            if(nums[fast] > nums[slow]){
                ++ slow;
                nums[slow] = nums[fast];
            }
            ++fast;
        }

        return slow+1;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        slow,fast=0,1
        sz=len(nums)
        while fast < sz:
            if nums[fast] != nums[slow]:
                slow+=1
                nums[slow] = nums[fast]
            fast+=1

        return slow+1

        """
        slow, fast = 0, 1
        while fast < len(nums):
            if nums[fast] != nums[slow]:
                slow += 1 #move from original position
                nums[slow] = nums[fast]
            fast += 1
        
        return slow+1
        """